aspnetcore/fundamentals/minimal-apis/includes/middleware8.md

```diff
@@ -77,4 +77,55 @@ app.Run(context =>
 
 Terminal middleware is middleware that runs if no endpoint handles the request.
 
+## Antiforgery middleware
+
+Call [AddAntiforgery](/dotnet/api/microsoft.extensions.dependencyinjection.antiforgeryservicecollectionextensions.addantiforgery) <!-- <xref:Microsoft.Extensions.DependencyInjection.AntiforgeryServiceCollectionExtensions.AddAntiforgery(IServiceCollection)> --> to add middleware for validating antiforgery tokens. Antiforgery tokens are used to mitigate [cross-site request forgery attacks](xref:security/anti-request-forgery).
```

Comment:
Calling `AddAntiforgery` won't add the middleware. It'll register the antiforgery services in DI. Then, `WebApplicationBuilder` will automatically add the middleware when it sees that antiforgery has been registered in the DI container.

```csharp
var builder = WebApplication.CreateBuilder();

builder.Services.AddAntiforgery(); // Adds anti-forgery services to DI

var app = builder.Build();

app.UseAntiforgery(); // User doesn't need to add this, the "app" will do it automatically

app.Run();
```